### PR TITLE
Documentation review

### PR DESCRIPTION
**Pull request by coot**
I reviewed up to Chapter 5 (including it).

### CODE REVIEW

File: doc/network.tex [hunk 1]
```diff
@@ -1576,10 +1622,10 @@ \subsection{Fairness and Flow-Control in the Multiplexer}
 the mini protocol and the ingress of the multiplexer.
 The mini protocol will block when it sends a message and the buffer is full.
 
-A concrete implementation of the a multiplexer may use a variety of data structures and heuristics to
+A concrete implementation of a multiplexer may use a variety of data structures and heuristics to
 yield the overall best efficiency.
 For example, although, the multiplexing protocol itself is agnostic to the underlying structure of
-the data, the multiplexer may try to avoid to split small mini protocol messages into two segments.
+the data, the multiplexer may in the future try to avoid to split small mini protocol messages into two segments.
```
Comment (coot) [inline doc/network.tex]: @karknu fix me if I am wrong here

Comment (MarcFontaine) [reply doc/network.tex]: The intended interpretation is that there are more details that a possible implementation may want to deal with, but that do not show up in the document. it is not specifically about the existing implementation

Comment (coot) [reply doc/network.tex]: If we joined SDUs from different mini-protocols, then any implementation would need to deal with it.

**Comment by coot** [comment]
Thanks @MarcFontaine!

**Pull request closed without merging**